data_map.go

```
@@ -1,12 +1,14 @@
 package mmdbwriter
 
 import (
+	"math"
 	"reflect"
 
 	"github.com/maxmind/mmdbwriter/v2/mmdbtype"
 )
 
 type dataMapKey string
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofumpt)